test/integration.spec.coffee

```diff
@@ -275,38 +276,39 @@ describe 'Product import integration tests', ->
         key: 'enum-new-key'
         label: 'enum-new-key'
 
-    newEnumKeyAttr2 =
```

Comment:
This test tried to save two attributes with the same name `sample_enum_attribute` and then read them both from the API. In this PR we filter out all duplicate attributes so only the first attribute got saved and thus this test was not working anymore.